src/main/java/callhub/connect/drivers/MessageWebSocketAccess.java

```diff
@@ -1,24 +1,43 @@
 package callhub.connect.drivers;
+import com.google.gson.Gson;
```

Comment:
just curious why did u pick this package for converting to json object

Reply:
I think gson is popular way for java
(and google #1)